index.html

```diff
@@ -25,7 +25,7 @@ <h1>Hello, Welcome to my Portfolio!</h1>
       <h1 class="project-tile">My Projects</h1>
       <a id="profile-link" target="_blank" href="https://github.com/hagan199">sample project</a>
     </section>
-
+<!---Contact-->
```

Comment:
Kindly avoid leaving commented out code in your code-base. This makes your work untidy and unprofessional. Especially comments coming from copied templates. 😸